packages/core/src/theme.js

```diff
@@ -312,9 +163,7 @@ const theme = {
   letterSpacings,
   regular,
   bold,
-  textStyles,
   colors,
-  colorStyles,
```

Comment:
Ensure removing these would not be a breaking change to anyone using the design system